templates/terraform/examples/app_engine_standard_app_version.tf.erb

```diff
@@ -0,0 +1,23 @@
+resource "google_project" "my_project" {
+  name       = "tf-test-project"
+  project_id = "<%= ctx[:vars]['project_id'] %>"
+  org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_app_engine_application" "app" {
+  project     = "${google_project.my_project.project_id}"
+  location_id = "us-central"
+}
+
+
+resource "google_app_engine_app_version" "<%= ctx[:primary_resource_id] %>" {
+  version_id = "v31"
+  project = "${google_app_engine_application.app.project}"
+  service = "default"
+  runtime = "nodejs10"
+  api_config = {}
```

Comment:
Make sure that fields that aren't being added right now aren't included, they'll cause the generated test to fail.